typescript-json-schema.ts

```diff
@@ -70,6 +70,7 @@ export type Definition = {
     $schema?: string,
     $id?: string,
     description?: string,
+    examples?: string[],
```

Comment:
Does an example always have to be a string?

Reply:
Doesn't seem to make any difference. Can resubmit with `any[]`

> There are no restrictions placed on the values within the array...

 > This keyword can be used to provide sample JSON values associated
   with a particular schema, for the purpose of illustrating usage.